src/index.js

```diff
@@ -22,7 +22,7 @@ function createContext(parentContext = {}) {
   return { ...parentContext };
 }
 
-export function createElement(elementName, attributes, ...children) {
+function createElement(elementName, attributes, ...children) {
```

Comment:
For discussion. Do you think we should keep named exports? Perhaps some people may prefer it over the default one.

Reply:
Take a look at the PR description.  Leaving these in place causes eslint to complain because trying to import named exports as members of the default export is a common mistake.  This rule is turned on in airbnb, so I think we probably want to avoid causing it to fail.

<img width="620" alt="screen shot 2018-09-17 at 17 09 49" src="https://user-images.githubusercontent.com/542836/45635437-e0900700-ba9c-11e8-9b9c-3dd8c17bdc4e.png">

Reply:
Sure, let's follow the default configuration 👍 We can always override it in the future if necessary.